joss_paper/paper.md

```diff
@@ -0,0 +1,458 @@
+  index: 33
+- name: Center for Astrophysics | Harvard & Smithsonian
+  index: 34
+- name: National Solar Observatory
+  index: 35
+- name: Royal Observatory of Belgium
+  index: 36
+- name: CIEMAT, Astroparticle physics, Madrid, Spain.
+  index: 37
+- name: Lockheed Martin Solar and Astrophysics Laboratory
+  index: 38
+- name: Department of Physics & Astronomy, Michigan State University, 567 Wilson Road, East Lansing, MI 48824, USA
+  index: 39
+- name: Department of Astronomy, University of Michigan, 1085 South University Avenue, Ann Arbor, MI 48109, USA
+  index: 40
+- name: Center for Computational Astrophysics, Flatiron Institute, 162 Fifth Ave, New York, NY 10010, USA
+  index: 41
+- name: Saw Swee Hock School of Public Health, National University Health System, National University of Singapore, Singapore
+  index: 42
+- name: International Institute of Information Technology, Hyderabad
+  index: 43
+- name: Indian Institute of Astrophysics, Bangalore
+  index: 44
+- name: University of Massachusetts, Amherst
+  index: 45
+- name: Indian Institute of Technology, Kharagpur
+  index: 46
+- name: SP2RC, School of Mathematics and Statistics, University of Sheffield, Sheffield, S3 7RH, UK
+  index: 47
+- name: Leibniz Institute for Astrophysics Potsdam, Germany
+  index: 48
+- name: Department of Chemical Engineering, Indian Institute of Technology Kharagpur, 721302, India
+  index: 49
+- name: Rosseland Centre for Solar Physics, University of Oslo, P.O. Box 1029 Blindern, NO-0315 Oslo, Norway
+  index: 50
+- name: Institute of Theoretical Astrophysics, University of Oslo, P.O. Box 1029 Blindern, NO-0315 Oslo, Norway
+  index: 51
+- name: Trinity College Dublin, Ireland
+  index: 52
+date: 3 October 2019
+bibliography: paper.bib
+aas-doi: 10.3847/1538-4357/ab4f7a
+aas-journal: The Astrophysical Journal
+---
+
+# Summary
+
+The Sun, our nearest star, is a local laboratory for studying universal physical processes. Solar physics as a discipline includes studying the Sun both as a star and as the primary driver of space weather throughout the heliosphere. Due to the Sun's proximity, the temporal and spatial resolution of solar observations are orders of magnitude larger than those of other stars. This leads to significant differences in the data-analysis software needs of solar physicists compared with astrophysicists.
+
+The `sunpy` Python package is a community-developed, free, and open-source solar data analysis environment for Python that provides a comprehensive set of tools for performing common tasks in solar data analysis. It is managed by the SunPy Project, an organization that facilitates and promotes the use of open development and open source packages like `sunpy` through community engagement and tools such as [GitHub](https://github.com/sunpy/sunpy/), [mailing lists](https://groups.google.com/forum/#!forum/sunpy), and [matrix](https://matrix.to/#/#sunpy:openastronomy.org).
```

Comment:
Removed the end of the first sentence (starting with "that provides...") as it is redundant. 
```suggestion
The `sunpy` Python package is a community-developed, free, and open-source solar data analysis environment for Python. It is managed by the SunPy Project, an organization that facilitates and promotes the use of open development and open source packages like `sunpy` through community engagement and tools such as [GitHub](https://github.com/sunpy/sunpy/), [mailing lists](https://groups.google.com/forum/#!forum/sunpy), and [matrix](https://matrix.to/#/#sunpy:openastronomy.org).
```